packages/web/src/components/buy-audio-modal/components/AudioAmountPicker.tsx

```diff
@@ -109,46 +104,44 @@ export const AudioAmountPicker = ({
     }
   }, [isCustomAmountInputVisible, customAmountRef])
 
-  const id = useMemo(() => uniqueId(), [])
-
   return (
     <>
       {!isCustomAmountInputVisible ? (
         <AmountPreview amount={presetAmount} />
       ) : null}
-      <RadioButtonGroup
-        aria-labelledby={`audioAmountPicker-label-${id}`}
-        className={styles.presetAmountButtons}
-        name='AmountPicker'
+      <RadioGroup
+        direction='row'
+        wrap='wrap'
+        gap='s'
+        aria-labelledby={'audioAmountPicker-label'}
+        name='amount'
         value={value}
         onChange={handleChange}
       >
-        <div id={`audioAmountPicker-label-${id}`} className={styles.label}>
+        <div id={'audioAmountPicker-label'} className={styles.label}>
```

Comment:
Did you drop the `id` on purpose?

Reply:
yeah basically not needed cause this modal is a singleton, so no chance there are two divs with the same id

Reply:
The modal is, but is this component forever going to be? It is right now but I've always liked components to be built without the singleton assumption